### DISCUSSION TITLE
测试mmbench dev集为什么只评测 index < int(1e6) 的数据？

### DISCUSSION THREAD

**Discussion by JKBox** [open-compass/opencompass#209] [View all discussions]
测试mmbench dev集为什么只评测 index < int(1e6) 的数据？而不是完整的dev集？
https://github.com/InternLM/opencompass/blob/main/tools/eval_mmbench.py#L317C5-L317C47

**Comment by kennymckormick** [comment]
Hi, JKBox, 
Note that MMBench adopts a CircularEval strategy, which means one question will have N copies (N = number of choices) with different choice shifting. `data_main` includes the first copy of all questions. While evaluating, we will retrieve all copies of a question with the first copy index.

**Reply by tonysy** [reply]
Maybe we can add the inline comment for https://github.com/InternLM/opencompass/blob/main/tools/eval_mmbench.py#L317C5-L317C47 to avoid confusion
